README.md

```diff
@@ -103,108 +103,117 @@ To learn more about Istio, please refer to the
 
 ### Architecture
 
-The pods and services that make up the Istio control plane is the first part of
-the architecture that gets installed into Kubernetes Engine, at the time we
-install the BookInfo application we simultaneously install an istio service
-proxy alongside each micro service in the application. At this point we have
+The pods and services that make up the Istio control plane are the first components of
+the architecture that get installed into Kubernetes Engine. When we
+install the BookInfo application we simultaneously install an Istio service
+proxy alongside each microservice in the application. At this point we have
 our two tiers that make up the Istio architecture, our Control Plane and our
 Data Plane, and we have microservices to manage.
 
-In the diagram, note
-* all input and output from any BookInfo microservice goes through the service
-  proxy
-* each service proxy communicates with each other and the Control plane to
+In the diagram, note:
+* All input and output from any BookInfo microservice goes through the service proxy.
+* Each service proxy communicates with each other and the Control plane to
   implement the features of the service mesh, circuit breaking, discovery, etc.
-* the Mixer component of the Control Plane is the conduit for the telemetry
-  add-ons to get metrics from the service mesh
-* the Istio ingress component to provide external access to the mesh
-* the environment is setup in the Kubernetes Engine default network
+* The Mixer component of the Control Plane is the conduit for the telemetry
+  add-ons to get metrics from the service mesh.
+* The Istio ingress component provides external access to the mesh.
+* The environment is setup in the Kubernetes Engine default network.
 
 ![](./images/istio-gke-gce.png)
 
-
 ## Prerequisites
 
-A Google Cloud account and project is required for this.  Access to an existing Google Cloud
-project with the Kubernetes Engine service enabled If you do not have a Google Cloud account
-please signup for a free trial [here](https://cloud.google.com).
+A Google Cloud account and a project with billing enabled are required for this demo to function. If you do not have a Google Cloud account please sign up for a free trial [here](https://cloud.google.com).
+
+### Supported Operating Systems
+
+This demo can be run from MacOS, Linux, or, alternatively, directly from [Google Cloud Shell](https://cloud.google.com/shell/docs/). The latter option is the simplest as it only requires browser access to GCP and no additional software need be installed. Instructions for both alternatives can be found below.
```

Comment:
no additional software installation is required.

Reply:
k